src/Mod/Fem/InitGui.py

```diff
@@ -55,7 +55,7 @@
 
 
 # add FEM Gui unit tests
-FreeCAD.__unit_test__ += ["TestFemGui"]
+# FreeCAD.__unit_test__ += ["TestFemGui"]
```

Comment:
Can we annotate it with comment when and why it was disabled? I know that the information will be in git blame but it is always good to state stuff like that explicitly.

Reply:
I didn't really expect to merge this as is, I was hoping to find the actual problematic test, and this commit was me attempting to make absolutely sure that it really was the FEM tests that were causing the failure. But with #24935 I am no longer certain that it tells us that at all.